apps/src/code-studio/components/libraries/LibraryPublisher.jsx

```diff
@@ -205,11 +226,21 @@ export default class LibraryPublisher extends React.Component {
         {this.displayDescription()}
         <Heading2>{i18n.catProcedures()}</Heading2>
         {this.displayFunctions()}
-        <Button
-          style={{marginLeft: 0, marginTop: 20}}
-          onClick={this.publish}
-          text={i18n.publish()}
-        />
+        <div style={{position: 'relative'}}>
+          <Button
+            style={{marginTop: 20}}
+            onClick={this.publish}
+            text={alreadyPublished ? i18n.update() : i18n.publish()}
+          />
+          {alreadyPublished && (
+            <Button
+              style={{right: 0, marginTop: 20, position: 'absolute'}}
```

Comment:
Worth extracting these styles to a const object?

Reply:
👍